cmd/sonarExecuteScan.go

```diff
@@ -131,6 +161,43 @@ func runSonar(config sonarExecuteScanOptions, client piperhttp.Downloader, runne
 	return nil
 }
 
+// isInOptions returns true, if the given property is already provided in config.Options.
+func isInOptions(config sonarExecuteScanOptions, property string) bool {
+	return SliceUtils.ContainsStringPart(config.Options, property)
+}
+
+func addJacocoReportPaths() {
+	matches, err := doublestarGlob(jacocoReportGlob)
+	if err != nil {
+		log.Entry().Warnf("failed to glob for Jacoco report paths: %v", err)
+		return
+	}
+	if len(matches) > 0 {
+		sonar.addOption(coverageReportPaths + strings.Join(matches, ","))
+	}
+}
+
+func addJavaBinaries() {
+	pomFiles, err := doublestarGlob("**/pom.xml")
+	if err != nil {
+		log.Entry().Warnf("failed to glob for pom modules: %v", err)
+		return
+	}
+	var binaries []string
+
+	for _, pomFile := range pomFiles {
+		module := filepath.Dir(pomFile)
+		classesPath := filepath.Join(module, "target", "classes")
```

Comment:
You should also consider `target/test-classes/`

Reply:
Done.